about.html

```diff
@@ -0,0 +1,150 @@
+              <img class="exit-menu" src="./Assets/x-bold.svg" alt="Exit Icon" />
+              <li class="nav-link"><a href="./index.html">Home</a></li>
+            <li class="nav-link"><a href="#Recipes">Recipes</a></li>
+            <li class="nav-link"><a href="#Buy">Buy</a></li>
+            <li class="nav-link"><a href="#Sponsor">Sponsor</a></li>
+            <li class="nav-link"><a href="#News">News</a></li>
+            </ul>
+            <button type="button">CC Compaign</button>
+          </div>
+        </nav>
+      </header>
+
+      <section id="home" class="home home-about">
+        <p class="hello-about">"Hello! cooking lovers"</p>
+        <h1>
+          CREATIVE MOROCCAN KITCHEN (CUISINE) 
+          2023
+        </h1>
+        <hr>
+        <p class="home-description">
+            Moroccan cuisine features a wide range of ingredients, including Mediterranean fruits, vegetables, beef, goat, mutton, lamb, chicken, and seafood. Common flavorings include lemon pickle, argan oil. <span class="to-delete"> preserved butter (smen), and olive oil. Staple grains include rice and wheat, used in bread and couscous. Spices like cinnamon, cumin, turmeric, and saffron are used extensively, with 27 spices combined to create the Moroccan spice mixture ras el hanout. Moroccan cuisine features a wide range of ingredients, including Mediterranean fruits, vegetables, beef, goat, mutton, lamb, chicken, and seafood. Common flavorings include lemon pickle, argan oil, preserved butter (smen), and olive oil. Staple grains include rice and wheat, used in bread and couscous. Spices like cinnamon, cumin, turmeric, and saffron are used extensively, with 27 spices combined to create the Moroccan spice mixture ras el hanout. Common herbs include mint, parsley, and coriander. Common herbs include mint, parsley, and coriander.</span>
+        </p>
+        <div class="date">
+          <h3>Please feel free to contact us to ask about Moroccan Cuisine!</h3>
+          <p>otmaneechchafyky5@gmail.com</p>
+        </div>
+      </section>
+
+      <section class="take-logo">
+        <div class="logo-info">
+            <h4>Here is the Moroccan Kitchen's logo</h4>
+            <hr>
+            <p class="logo-info-para">
+                Big brands hire only the best (or at least, the most expensive) designers, so more often than not. <span class="to-delete"> they have some of the highest quality logos around.
+                Big brands. Big brands hire only the best (or at least, the most expensive) designers, so more often than not, they have some of the highest quality logos around.</span>
+            </p>
+        </div>
+        <a class="logo-container" href="#">
+            <img src="./Assets/logo.svg" alt="Logo Icon" />
+            <h3>The Moroccan Kitchen.</h3>
+        </a>
+      </section>
+
+      <hr class="section-devider">
+
+      <section class="events">
+        <div class="logo-info">
+            <h4>See our latest events</h4>
```

Comment:
Kindly obey the header hierarchy (hx) rule throughout your project👍. every other section apart from the main section should begin with an `h2` and within the section, other headings should use the lower hierarchy accordingly (h3, h4, h5, etc)👍 This gives your project a higher semantic store and allows the screen readers have a better understanding of your page structure👍